pkg/policy/identity.go

```diff
@@ -28,6 +28,11 @@ import (
 	"github.com/cilium/cilium/pkg/logging/logfields"
 	"github.com/cilium/cilium/pkg/u8proto"
 
+	"encoding/json"
+	// TODO - uncomment this when we actually update the proxyMutator for IP Cache
+	//"github.com/cilium/cilium/pkg/envoy/api"
```

Comment:
FYI, that import path will change with https://github.com/cilium/cilium/pull/2781.